packages/trader/src/Stores/Modules/Trading/Helpers/end-time.ts

```diff
@@ -38,8 +38,8 @@ export const getSelectedTime = (
 
 export const getBoundaries = (
     server_time: TTime['server_time'],
-    market_open_times: TTime['market_open_times'],
-    market_close_times: TTime['market_close_times']
+    market_open_times: moment.Moment[],
```

Comment:
@maryia-deriv  In this case type arguments is `moment.Moment[]`, because `getBoundaries` is used only in one file (_packages/trader/src/Modules/Trading/Components/Form/TimePicker/trading-time-picker.tsx_), where it accepts params with type `moment.Moment[]`
<img width="1080" alt="Screenshot 2023-09-08 at 4 03 56 PM" src="https://github.com/binary-com/deriv-app/assets/121025168/a092c8b8-de88-43ba-b404-f9d578f3b312">